starter-code/js/index.js

```diff
@@ -11,7 +11,33 @@ function updatePriceByProduct(productPrice, index){
 }
 
 function getTotalPrice() {
+  var totalCost = 0;
+  for (var x = 0; x < document.querySelectorAll('.product').length ; x ++) {
 
+  var unitValue = Number(document.querySelectorAll('.cost')[x].innerHTML);
+  var totalUnits = Number(document.querySelectorAll('input')[x].value);
+
+  var totalCostProduct = unitValue * totalUnits;
+
+  var totalCost = totalCost + totalCostProduct;
+
+
+  document.getElementsByClassName('total-cost-product')[x].innerHTML = "$" + totalCostProduct;
+  }
+
+  var heading2 = document.querySelectorAll('h2')[0];
+    if (document.querySelectorAll('h2')[0])  {
+        document.getElementById('total-price-sentence').removeChild(heading2);  
```

Comment:
You don't need to remove this. Instead of creating a new element (what you are doing below) you can modify the .innerText of the element that you already have